docs/getting_started.md

```diff
@@ -1,40 +1,170 @@
- 
-## Directory system
 
-On the left hand side of your screen is a directory system. Make a new directory for yourself using your name as a label. Right click and select, ‘New Folder’. Moving forward always work within this named directory.
+!!! tip
+    Sometimes you need to re-start your notebook: your screen freezes, you made a mistake, you forgot to filter your data, you want to re-run your notebook to check for reproducibility, etc. On the top banner, select 'Kernel' and from the drop down chose the option that fits your needs. You can restart the kernel and clear all ouput to re-run your notebook. Text in the cells will be saved, but will need to be re-executed.
 
-For organization and reproducibility everytime you start a new data analysis project, set up a directory system for your analysis. Many people use the same structure for every data project. For example:
- 
--        Main directory – memorable title of your data project 
--        Subdirectories – 
-data (for raw data) 
-notebooks (to save your code and documentation)
-output (direct output generated from your code) 
-Figures (figures generated from your code)
-src (if you are using scripts) 
--        README – Always include a README within each directory that describes the purpose of the directory and a description of what files are found within the directory.
+## Importing data as flat files
 
-## Computational notebooks (link to learning hub) 
+Data can be uploaded into your directory for analysis or to share with other users. From your local computer drag and drop the data file into your directory on the server. 
 
-The CIAP provides access to computational notebooks, Jupyter Notebooks, which allow users to combine documentation, code, and results into one report. Code and results are recorded as the code is executed promoting reproducibility, communication of analysis and how results were obtained.  Jupyter Notebooks sit alongside data, curated and custom ILCI tools, and funded computation resources. You have everything you need in one place.
- 
-First navigate to your project/notebook directory. Create a notebook in the Launcher by selecting either the ‘R’ or ‘Python’ icon, depending upon which language you use. You can also create a notebook by right clicking in the directory window or by navigating to “File” and “New” Notebook. When creating a notebook from the Launcher the programming language is preset for you. Otherwise you will need to navigate to the top right corner of the notebook if you’d like to use R for coding. Click on ‘Python 3 (ipykernel)’ and in the drop down menu select ‘R’. 
+![Flat file](img/getting_started/data.png){ align=left }
 
-Sava and name your notebook. You can do this via File -> Save Notebook, or push the save button on the top left hand corner of your notebook. You can rename by right clicking on the untitled notebook. 
- 
-Name your notebook something meaningful that will help you to identify the analysis later. 
 
+!!! tip
```

Comment:
I like this idea of "peppering" in "tip blocks" throughout this page to link out to more granular docs.